src/index.js

```diff
@@ -302,10 +302,34 @@ const addLang = (tree, {rules: {lang}}) => {
   return tree;
 };
 
+const commentsFormatting = tree => {
+  tree.walk = walk;
+  tree.walk(node => {
+    if (typeof node === 'string' && /<!--[^]*-->/gm.test(node.trim())) {
+      const originalComments = node.trim();
+      const content = originalComments.replace(/^<!--/g, '').replace(/-->$/, '');
```

Comment:
Here, it seems to me that a regular expression that takes into account a multiline is better - `<!--([\s\S]*?)-->` content will match

Reply:
I will check this.

Reply:
done

Reply:
Pay attention to this post here is a coincidence and there will be our content, and we do not need to `replace`

Reply:
I didnt get this 🔼